### PR TITLE
Pushing changes for the ability to remove the dataset after import

### PR DESCRIPTION
**Pull request by colin-kidwell**

### CODE REVIEW

File: panels/nerf_3dview_panels/dataset_panel.py [hunk 1]
```diff
@@ -33,15 +37,20 @@ def unregister(cls):
...
+        bpy.utils.unregister_class(RemoveNeRFDatasetOperator)
         cls.remove_observers()
 
 
 
-    def draw(self, context):
-        """Draw the panel with corresponding properties and operators."""
-
-        layout = self.layout
+    def dataset_section(self, context, layout, ui_props):
```
Comment (JamesPerlman) [inline panels/nerf_3dview_panels/dataset_panel.py:45]: why was this changed?

File: panels/nerf_panel_operators/import_dataset_operator.py [hunk 2]
```diff
@@ -45,13 +50,18 @@ class ImportNeRFDatasetOperator(bpy.types.Operator):
 
     @classmethod
     def poll(cls, context):
-        return NeRFManager.can_import()
+        return NeRFManager.can_import
```
Comment (JamesPerlman) [inline panels/nerf_panel_operators/import_dataset_operator.py:53]: as far as i can tell, this is still a function (not a property).  It may evaluate to be truthy here, but possible because the `NeRFManager.can_import` method is not `None` - until you overwrite it with a `True` or `False` value

File: panels/nerf_panel_operators/import_dataset_operator.py [hunk 3]
```diff
@@ -147,7 +157,8 @@ def execute(self, context):
             sensor_width = cam_data.sensor_width
             cam_data.lens = sensor_width / cam_w * fl_x
 
-        context.view_layer.objects.active = nerf_obj
+        context.scene.nerf_panel_ui_props.imported_dataset_path = self.filepath
+        NeRFManager.can_import = False
```
Comment (JamesPerlman) [inline panels/nerf_panel_operators/import_dataset_operator.py:161]: I wouldn't mind changing this to a settable value, but right now it's still a method not a property

File: panels/nerf_3dview_panels/dataset_panel.py (hunk 1 above)
Comment (colin-kidwell) [reply panels/nerf_3dview_panels/dataset_panel.py:45]: To try and see how I can alter the training panel. Chances are that defining can be reverted

File: panels/nerf_panel_operators/import_dataset_operator.py [hunk 4]
```diff
@@ -158,14 +169,45 @@ def invoke(self, context, event):
...
+        context.scene.nerf_panel_ui_props.imported_dataset_path = ""
+        NeRFManager.can_import = True
+        
+        # Select and delete all objects in the scene
+        bpy.ops.object.select_all(action='SELECT')
+        bpy.ops.object.delete(use_global=False)
+
+        #Clear Recursive Datablocks 
+        #This is so the file doesnt have unnecessary data attached to it. 
+        bpy.ops.outliner.orphans_purge(do_local_ids=True, do_linked_ids=True, do_recursive=True)
```
Comment (JamesPerlman) [inline panels/nerf_panel_operators/import_dataset_operator.py:187]: This part looks good, i like this.

**Review by JamesPerlman — CHANGES_REQUESTED** [review]

File: panels/nerf_panel_operators/import_dataset_operator.py (hunk 2 above)
Comment (colin-kidwell) [reply panels/nerf_panel_operators/import_dataset_operator.py:53]: After import i set can import to true and after removal i set it to false

File: panels/nerf_panel_operators/import_dataset_operator.py (hunk 3 above)
Comment (colin-kidwell) [reply panels/nerf_panel_operators/import_dataset_operator.py:161]: it worked when i was working with it because like you said you can set it to true or false because initially it isn't set to a value. Does this make sense? If it doesn't lmk

**Review by JamesPerlman — APPROVED** [review]
